src/utils/propertyHelper.test.js

```diff
@@ -109,3 +109,39 @@ describe("isObjectContainsPropertyWithValue", () => {
 		expect(result, "to be false");
 	});
 });
+
+describe("isObjectContainsPropertyWithAnyValue", () => {
+	it("Returns true if object contains specified property with any value", () => {
+		const obj = {
+			prop1: {
+				a: {
+					m: "Hello",
+				},
+			},
+			prop2: {
+				m: "Hello Bobbina",
```

Comment:
Who is Bobbina?  ;)

Reply:
Female adaptation of Bob based on slavic languages I guess :)